tests/options/attributeSeparator/always/always.test.ts

```diff
@@ -11,6 +11,7 @@ describe('Options', () => {
 			const actual: string = format(code, {
 				parser: 'pug' as any,
 				plugins: [plugin],
+				printWidth: 80,
```

Comment:
Why is this needed? I mean `80` is the default
If it is intended cause any reason, you could add a comment, so we also now in 100+ years (if pug is also then a thing)

Reply:
I added it explicitly because the test runs against that `printWidth`. Kinda the same reason why you added `endOfLine: 'lf'` ;)

Happy to remove it though

Reply:
We should add a comment, and yes I will also add a comment on the `endOfLine: 'lf'` thingy later (when my working day has ended)